src/ProductCard.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components'
+
+function ProductCard(props) {
+  return (
+    <Wrapper>
+      <Image src={props.image} alt="A bottle of Lakritzel"></Image>
```

Comment:
You can write it like this:
`<Image src={props.image} alt="A bottle of Lakritzel" />`